octokit/request.go

```diff
@@ -35,6 +35,12 @@ func (r *Request) Get(output interface{}) (*Response, error) {
 	return r.createResponse(r.Request.Get(), output)
 }
 
+// Get sends a GET request through the given client and returns the response
+// and any associated errors. It does not fill in a response body.
+func (r *Request) GetBodyless() (*Response, error) {
```

Comment:
Not seeing why we need a new method here. Can we not make `Get` handle those types of responses?